<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStackedWidget#stackedWidget QPushButton
{
	border:none;
	background-color: 
	border-radius:5px;
	color:white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QPushButton" name="checkBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>查看客房</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manageBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>客房管理</string>
      </property>
     </widget>
     <widget class="QLabel" name="maintitle">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 26pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>酒店客房管理系统</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
